packages/ffmpeg.rb

```diff
@@ -41,6 +28,7 @@ class Ffmpeg < Package
   depends_on 'vidstab'
 
   def self.build
+    ENV['TMPDIR'] = "#{CREW_PREFIX}/tmp"
```

Comment:
This line is unneccessary.  The line below uses a completely different temp directory.